webpack.config.js

```diff
@@ -1,20 +1,30 @@
-const path = require("path")
-const HtmlWebpackPlugin = require("html-webpack-plugin")
-const outputDir = path.join(__dirname, "build/")
+const path = require("path");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const FaviconWebpackPlugin = require("favicons-webpack-plugin");
+const outputDir = path.join(__dirname, "build/");
 
-const isProd = process.env.NODE_ENV === "production"
+const isProd = process.env.NODE_ENV === "production";
 
 module.exports = {
   entry: "./src/Index.bs.js",
   mode: isProd ? "production" : "development",
   output: {
     path: outputDir,
+    publicPath: "/",
     filename: "Index.js"
   },
   plugins: [
     new HtmlWebpackPlugin({
       template: "src/index.html",
-      inject: false
+      favicon: "src/favicon.ico",
+      inject: true
```

Comment:
Why would you ever want `inject` to be `false`, like it was before?

Reply:
Good question - my webpack knowledge is rather limited, but my guess is that it's fine for simple cases like the default bucklescript template, where it's enough to include some static file.